### PR TITLE
docs: update sample-lnd.conf

### PR DESCRIPTION
**Pull request by Roasbeef**
In this commit, we update the `sample-lnd.conf` example config file to
be up to date with all the new configuration parameters we've added over
the past few release.

### CODE REVIEW

File: sample-lnd.conf [hunk 1]
```diff
@@ -436,3 +601,94 @@ litecoin.node=ltcd
...
+ 
+; Whether we intend to skip TLS verification
+; db.etcd.insecure_skip_verify=true                         
+ 
+; Whether to collect etcd commit stats.
+; db.etcd.collect_stats=true                                 
+
+[bolt]
+; If true, prevents the database from syncing its freelist to disk. 
+db.bolt.nofreelistsync
```
Comment (guggero) [inline sample-lnd.conf:694]: Boolean flags need a value in the config file.

Comment (guggero) [reply sample-lnd.conf:694]: Also, is this not commented out on purpose?

File: sample-lnd.conf [hunk 2]
```diff
@@ -436,3 +601,94 @@ litecoin.node=ltcd
...
+; roughly 2Kb. (default: 20000)
+; caches.channel-cache-size=9000000
+
+[protocol]
+; If set, then lnd will create and accept requests for channels larger than 0.16
+; BTC
+; protocol.wumbo-channels=1                               
+
+; Set to enable experimental support for anchor commitments, won't work with watchtowers yet.
+; protocol.anchors
```
Comment (guggero) [inline sample-lnd.conf:663]: Boolean flags need a value in the config file.

File: sample-lnd.conf [hunk 3]
```diff
@@ -356,6 +492,35 @@ litecoin.node=ltcd
...
+; The password used to arrive at the HashedControlPassword for the control port.
+; If provided, the HASHEDPASSWORD authentication method will be used instead of
+; the SAFECOOKIE one.
+; tor.password=plsdonthackme                                         
+ 
+; Automatically set up a v2 onion service to listen for inbound connections
+; tor.v2=1                                                
+ 
+; Automatically set up a v3 onion service to listen for inbound connections
+; tor.v3=1
```
Comment (guggero) [inline sample-lnd.conf:516]: I think we should use `true` or `false` for boolean flags everywhere to make the config more consistent. At least there was a PR a while ago that did it for the then existing values.

File: sample-lnd.conf [hunk 4]
```diff
@@ -339,7 +450,32 @@ litecoin.node=ltcd
...
+; created by the autopilot agent must have. (default: 1)
+; autopilot.minconfs=2
+
+; The confirmation target (in blocks) for channels opened by autopilot. (default:
+; 3)
+; autopilot.conftarget=2
+
 [tor]
+; Allow outbound and inbound connections to be routed through Tor
+; tor.active
```
Comment (guggero) [inline sample-lnd.conf:477]: Boolean flags need a value in the config file.

**Review by guggero — COMMENTED** [review]
Great PR, shows how many new features we added recently :100: 
Found a few more nits, otherwise looks good to me!

File: sample-lnd.conf (hunk 3 above)
Comment (Roasbeef) [reply sample-lnd.conf]: Updated to be consistent everywhere!

**Pull request merged**